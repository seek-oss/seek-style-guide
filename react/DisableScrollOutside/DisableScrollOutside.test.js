import { shallow } from 'enzyme';
import React from 'react';

import DisableScrollOutside from './DisableScrollOutside';

describe('DisableScrollOutside', () => {
  it('should render without props', () => {
    const wrapper = shallow(<DisableScrollOutside />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should render with children', () => {
    const wrapper = shallow(<DisableScrollOutside children={<p>Children</p>} />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should render with body lock', () => {
    const wrapper = shallow(<DisableScrollOutside children="Children" shouldLockBody />);
    expect(wrapper).toMatchSnapshot();
  });
});
