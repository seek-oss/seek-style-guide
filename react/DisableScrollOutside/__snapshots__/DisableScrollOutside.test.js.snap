// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DisableScrollOutside should render with body lock 1`] = `
<div>
  Children
  <Component />
</div>
`;

exports[`DisableScrollOutside should render with children 1`] = `
<div>
  <p>
    Children
  </p>
</div>
`;

exports[`DisableScrollOutside should render without props 1`] = `<div />`;
